src/services/TestResultsService.ts

```diff
@@ -271,14 +274,20 @@ export class TestResultsService {
         validation = testTypesSchemaGroup1.validate(testType, {context});
       } else if (TEST_TYPES_GROUP2.includes(testType.testTypeId)) {
         validation = testTypesSchemaGroup2.validate(testType, {context});
-      } else if (TEST_TYPES_GROUP3_4_5_10.includes(testType.testTypeId)) {
-        validation = testTypesSchemaGroup3And4And5And10.validate(testType, {context});
-      } else if (TEST_TYPES_GROUP6_7_8.includes(testType.testTypeId)) {
-        validation = testTypesSchemaGroup6And7And8.validate(testType, {context});
-      } else if (TEST_TYPES_GROUP9_11.includes(testType.testTypeId)) {
-        validation = testTypesSchemaGroup9And11.validate(testType, {context});
-      } else if (TEST_TYPES_GROUP12_13.includes(testType.testTypeId)) {
-        validation = testTypesSchemaGroup12And13.validate(testType, {context});
+      } else if (TEST_TYPES_GROUP3_4_8.includes(testType.testTypeId)) {
```

Comment:
This would be the other possible place to put "this group represents XXXX" e.g. "this group represents hgv LEC tests"

Reply:
done